consensus/interface.go

```diff
@@ -152,8 +152,8 @@ type ScheduledProcessor interface {
 // SignatureHandler defines the behaviour of a component that handles signatures in consensus
 type SignatureHandler interface {
 	Reset(pubKeys []string) error
-	CreateSignatureShare(msg []byte, index uint16, epoch uint32) ([]byte, error)
-	CreateSignatureShareWithPrivateKey(message []byte, index uint16, epoch uint32, privateKeyBytes []byte) ([]byte, error)
+	CreateSignatureShareUsingPublicKey(message []byte, index uint16, epoch uint32, publicKeyBytes []byte) ([]byte, error)
```

Comment:
can you rename to CreateSignatureShareForPublicKey and CreateSignatureForPublicKey ?

Reply:
Renamed both methods to `CreateSignatureShareForPublicKey` and `CreateSignatureForPublicKey`